src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -78,4 +97,46 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           + " " + lineContent);
     }
   }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long lectureId = Long.parseLong(request.getParameter(PARAM_LECTURE_ID));
+    Key lecture = KeyFactory.createKey(PARAM_LECTURE, lectureId);
+
+    Filter lectureFilter =
+        new FilterPredicate(TranscriptLine.PROP_LECTURE, FilterOperator.EQUAL, lecture);
+
+    Query query = new Query(TranscriptLine.ENTITY_KIND)
+                      .setFilter(lectureFilter)
+                      .addSort(TranscriptLine.PROP_START, SortDirection.ASCENDING);
+    PreparedQuery pq = datastore.prepare(query);
+
+    ImmutableList.Builder<TranscriptLine> lineBuilder = new ImmutableList.Builder<>();
+    for (Entity entity : pq.asQueryResultIterable()) {
+      lineBuilder.add(TranscriptLine.fromEntity(entity));
+    }
+    ImmutableList<TranscriptLine> lines = lineBuilder.build();
+
+    Gson gson = new Gson();
+    response.setContentType("application/json;");
+    response.getWriter().println(gson.toJson(lines));
+  }
+
+  /**
+   * Creates a line entity using the attributes from {@code node} and {@code lectureId}.
+   */
+  private Entity createLineEntity(Node node, long lectureId) {
+    Element element = (Element) node;
+    String lineContent = node.getTextContent();
+    String lineStart = element.getAttribute(START_ATTRIBUTE);
+    String lineDuration = element.getAttribute(DURATION_ATTRIBUTE);
+
+    Entity lineEntity = new Entity(TranscriptLine.ENTITY_KIND);
+    lineEntity.setProperty(
+        TranscriptLine.PROP_LECTURE, KeyFactory.createKey(PARAM_LECTURE, lectureId));
```

Comment:
Add a TODO to change PARAM_LECTURE to Lecture.ENTITY_KIND once lecture stuff is merged with this